components/drawer/__tests__/__snapshots__/Drawer.test.js.snap

```diff
@@ -12,7 +12,7 @@ exports[`Drawer className is test_drawer 1`] = `
     />
     <div
       class="ant-drawer-content-wrapper"
-      style="transform:translateX(100%);width:256px"
+      style="transform:translateX(100%);-ms-transform:translateX(100%);width:256px"
```

Comment:
Why ant-drawer-content-wrapper is fixed to `256px`? I can not change it with `style` or `drawerStyle`.